flaggers/flagger.py

```diff
@@ -35,6 +35,14 @@ class Flags(Enum):
   SCHEDULE_STATUS_NULL = auto()
   TRIP_ID_NULL = auto()
 
+  #Unobserved stop flag
```

Comment:
Nitpick: you don't have to place these comments, the flag names are really obvious.